tox.ini

```diff
@@ -6,7 +6,7 @@ commands =
     python setup.py nosetests {posargs}
 deps =
     nose
-    mg34x: PyMongo>=3.4,<3.5
-    mg3x: PyMongo>=3.0,<3.7
+    mg34: pymongo>=3.4,<3.5
```

Comment:
Since you've changed this to `mg34`, you have  to change the global var above to `PYMONGO_3_4=3.4`.

This is why [the latest build](https://travis-ci.org/MongoEngine/mongoengine/jobs/553469862) has failed with `ERROR: unknown environment 'py27-mg34x'`.